File: kodtaz/#Zoner2.pl
sub EVENT_SPAWN {
  	$x = $npc->GetX();
   	$y = $npc->GetY();
	quest::set_proximity($x - 10, $x + 10, $y - 10, $y + 10);
}

sub EVENT_ENTER {
	$InInstanceIkky2 = quest::GetInstanceID("ikkinz",1);

    if($InInstanceIkky2 > 0){
		quest::MovePCInstance(294,$InInstanceIkky2,-157,27,-2);
	}
        else {
		$client->Message(13, "You are not a part of an instance!");
	}
}

File: kodtaz/#Zoner2.lua
function event_spawn(e)
  local x = e.self:GetX()
  local y = e.self:GetY()
  eq.set_proximity(x - 10, x + 10, y - 10,  y + 10)
end

function event_enter(e)
  local dz_version = instance_version.ikkinz_chambers_of_twin_struggles
  e.other:MovePCDynamicZone("ikkinz", dz_version, false) -- Zone: ikkinz group 2 version
end
